tests/integration/components/light-table-test.js

```diff
@@ -174,24 +170,23 @@ test('passed in components can have computed properties', function(assert) {
     {{/light-table}}
   `);
 
-  assert.equal(this.$('.custom-row').length, 3, 'three custom rows were rendered');
-  assert.equal(this.$('.custom-row.is-active').length, 0, 'none of the items are active');
+  assert.equal(findAll('.custom-row').length, 3, 'three custom rows were rendered');
+  assert.notOk(find('.custom-row.is-active'), 'none of the items are active');
 
   this.set('current', users[0]);
-
-  assert.ok(this.$('.custom-row:eq(0)').hasClass('is-active'), 'first custom row is active');
+  let [firstRow] = findAll('.custom-row');
+  assert.ok(hasClass(firstRow, 'is-active'), 'first custom row is active');
 
   this.set('current', users[2]);
-
-  assert.ok(this.$('.custom-row:eq(2)').hasClass('is-active'), 'third custom row is active');
+  let [,, thirdRow] = findAll('.custom-row');
```

Comment:
I started with the array indexing, but then `ember-suave` complained about it and I wasn't in the mood to investigate right now. will swap in `nth-child` selectors